Frameworks/CoreText/CTFontManager.mm

```diff
@@ -59,12 +59,11 @@ static bool __CTFontManagerUpdateWithFonts(CFArrayRef fontURLs, CTFontManagerSco
     woc::unique_cf<CFMutableArrayRef> fontDatas{ CFArrayCreateMutable(nullptr, count, &kCFTypeArrayCallBacks) };
     for (size_t i = 0; i < count; ++i) {
         NSData* data = [NSData dataWithContentsOfURL:static_cast<NSURL*>(CFArrayGetValueAtIndex(fontURLs, i))];
-        if (data != nullptr) {
-            CFArrayAppendValue(fontDatas.get(), (CFDataRef)data);
-        }
+        CFArrayAppendValue(fontDatas.get(), (CFDataRef)data);
     }
 
-    return SUCCEEDED(func(fontDatas.get(), errors));
+    // S_FALSE represents partial failure so cannot use SUCCEEDED macro
```

Comment:
It made more sense to use S_FALSE than E_* that might mean a more fatal failure than what we're experiencing